File: LV1/1.6/sumkindofproblem/sumkindofproblem.rb
n = gets.to_i

n.times do
  k, r = gets.split.map(&:to_i)
  sum_r = r * (r + 1) / 2
  puts "#{k} #{sum_r} #{2 * sum_r - r} #{2 * sum_r}"
end

File: LV1/1.6/sumkindofproblem/sumkindofproblem.go
package main

import "fmt"

func main() {
	var n, k, r int
	fmt.Scanf("%d", &n)

	for i := 0; i < n; i++ {
		fmt.Scanf("%d %d", &k, &r)
		sumR := r * (r + 1) / 2
		fmt.Println(k, sumR, 2*sumR-r, 2*sumR)
	}
}
